src/modules/functionsServer.ts

```diff
@@ -27,22 +25,63 @@ export const functionsServer = async (basePath: string, port: number) => {
     })
   )
 
-  console.log('')
-
   app.all('/*', async (req: Request, res: Response) => {
     const [endpoint] = req.params['0'].split('/')
 
-    const lambdaEvent = {
+    const queryStringParameters: { [key: string]: string } = {}
+
+    Object.entries(req.query).forEach(([key, value]) => {
+      if (value !== undefined) {
+        queryStringParameters[key] = value.toString()
+      }
+    })
+
+    const headers: { [key: string]: string } = {}
+
+    Object.entries(req.headers).forEach(([key, value]) => {
+      if (value !== undefined) {
+        if (Array.isArray(value)) {
+          headers[key] = value[value.length - 1]
+        } else {
+          headers[key] = value
+        }
+      }
+    })
+
+    const now = new Date()
+    const lambdaEvent: APIGatewayProxyEventV2 = {
```

Comment:
A good practice is to create utility functions, to keep the main code cleaner and easy to understand. Here you could create one, or when you build the querystring and headers.